uportal-war/src/main/java/net/sf/ehcache/distribution/jgroups/BootstrapRequest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ *  Copyright 2003-2010 Terracotta, Inc.
```

Comment:
I'm confused... this class exists within the `group: 'net.sf.ehcache', name: 'ehcache-jgroupsreplication', version: '1.7'` library that @ChristianMurphy linked?  If so, can we just add iti as a dependency?  (I assume there's a reason we aren't doing that, but it's unclear to me.)